google/datalab/bigquery/_table.py

```diff
@@ -388,7 +388,7 @@ def _init_job_from_response(self, response):
       job = _job.Job(job_id=response['jobReference']['jobId'], context=self._context)
     return job
 
-  def extract_async(self, destination, format='csv', csv_delimiter=',', csv_header=True,
+  def extract_async(self, destination, format='csv', csv_delimiter=None, csv_header=True,
```

Comment:
"defaults to ','" is different from "if None and format is 'csv', ',' is used". Perhaps I am nitpicking here. :)

It seems BigQuery should ignore csv_delimiter if format is not 'csv'. We could always pass None to BigQuery if format is not 'csv' to fix that. I am okay with either.